src/applications/vaos/sass/vaos.scss

```diff
@@ -101,4 +101,8 @@ body {
   .vaos__dynamic-font-size--h2 {
     font-size: 24px;
   }
-}
\ No newline at end of file
+}
+
```

Comment:
Slight concern here since this style will be added to all textarea widgets. However, this is localized for VAOS and "box-sizing: border-box" is defined on parent widgets but for some strange reason is not cascaded to this child widget.

Reply:
I think it's reasonable since it should be set to `border-box` for textarea widgets anyway. Weird it didn't cascade correctly though. Note that I couldn't reproduce the original issue locally but did confirm it to appear in the staging environment (somehow user agent stylesheet behaves differently between local and staging). This fix apply the styles at a higher priority so it should fix the issue though I can't really verify it locally.